install/installer/pkg/common/customize_test.go

```diff
@@ -521,13 +527,6 @@ func TestCustomizeEnvvar(t *testing.T) {
 
 			result := common.CustomizeEnvvar(ctx, testCase.Component, testCase.ExistingEnnvars)
 
-			sort.Slice(result, func(a, b int) bool {
```

Comment:
Removing this ensures that the order generated by the function matches the expectation